core_backend/tests/api/test_data_api.py

```diff
@@ -37,6 +43,145 @@ def utcnow(self) -> datetime:
         return self.date
 
 
+class TestUrgencyQueryDataAPI:
+    @pytest.fixture
+    async def user1_data(
+        self,
+        monkeypatch: pytest.MonkeyPatch,
+        asession: AsyncSession,
+        urgency_rules: int,
+    ) -> AsyncGenerator[None, None]:
+        dates = [
+            datetime.utcnow() - relativedelta(days=x) for x in range(N_DAYS_HISTORY)
```

Comment:
I get this warning on all `datetime.utcnow()`! :

> The method "utcnow" in class "datetime" is deprecated
>  Use timezone-aware objects to represent datetimes in UTC; e.g. by calling .now(datetime.UTC)

Reply:
Yep! I'll update it. I'm rebasing over Amir's change now.